docs/docs/getting-started/quickstart.md

```diff
@@ -1,58 +1,4 @@
-# Getting Started
```

Comment:
was removing all of this intentional?

Reply:
yes, the content has been split in two locations